### PR TITLE
doc: Add Datacake integration guide

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding the guide for creating a Webhook integration with Datacake.

#### Testing####
This integration was tested manually with TTS instance running locally.

#### Checklist

- [x ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/webhooks/webhook-templates/_index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: "Webhook Templates"
 description: ""
 summary: Webhook templates define a webhook integration that is not created (yet). Templates allows for using common values for many webhooks, such as a common base paths.
+weight: 1
```
Comment (johanstokking) [inline doc/content/integrations/webhooks/webhook-templates/_index.md:5]: why?

File: doc/content/integrations/webhooks/cloud-integrations/Datacake/Creating a Webhook on TTS/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (johanstokking) [inline doc/content/integrations/webhooks/cloud-integrations/Datacake/Creating a Webhook on TTS/_index.md:1]: Check folder structure; use lowercase, no spaces, prefer dashes over undercores in folder and filenames (except `_index` etc)

File: doc/content/integrations/webhooks/webhook-templates/_index.md (hunk 1 above)
Comment (nejraselimovic) [reply doc/content/integrations/webhooks/webhook-templates/_index.md:5]: I thought it made more sense to have the generic documentation listed before the platforms-specific one, but I can change this if you disagree

Comment (johanstokking) [reply doc/content/integrations/webhooks/webhook-templates/_index.md:5]: OK but that it would merely serve as an introduction?

Comment (nejraselimovic) [reply doc/content/integrations/webhooks/webhook-templates/_index.md:5]: When I put this weight like this, we have an Overview section where the whole generic process of creating a webhook is explained, then this section Webhook Templates, then the Cloud Integrations section. I don't see the Webhook Templates section being an introduction here, am I missing something here?
Screenshot from 2020-06-12 13-08-27

Comment (johanstokking) [reply doc/content/integrations/webhooks/webhook-templates/_index.md:5]: I see now, thanks, this looks good!

Sorry for the confusion

File: doc/content/integrations/webhooks/cloud-integrations/datacake/_index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+weight: 
+---
+
+[Datacake](https://datacake.co/) is a multi-purpose IoT platform, which provides the possibility of building custom IoT applications, without requiring programming skills. 
+
+<!--more-->
+
+## Requirements
+
+1. You need to have a user account on Datacake.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/datacake/_index.md:13]:
```suggestion
1. A user account on Datacake.
```

File: doc/content/integrations/webhooks/cloud-integrations/datacake/tts-setup/_index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+<!--more-->
+
+>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 
+
+Next, you need to add an **Authorization** header, whose value will consist of the word "Token" and your [API token](https://docs.datacake.de/api/generate-access-token) from Datacake.
+
+According to the **Webhook Settings** information that can be found in the **Configuration** tab on Datacake, the **Base URL** value needs to be `https://api.datacake.co/integrations/lorawan/tti/`.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/datacake/tts-setup/_index.md:17]:
```suggestion
Per the **Webhook Settings** information that can be found in the **Configuration** tab on Datacake, set the **Base URL** value to `https://api.datacake.co/integrations/lorawan/tti/`.
```

**Pull request merged**